front/src/hooks/useLoadAndSetGlobalPreferences.ts

```diff
@@ -23,16 +32,27 @@ export const useLoadAndSetGlobalPreferences = () => {
   const [runLocationAndShipmentQuery] = useLazyQuery<MultiBoxActionOptionsForLocationsTagsAndShipmentsQuery>(
     MULTI_BOX_ACTION_OPTIONS_FOR_LOCATIONS_TAGS_AND_SHIPMENTS_QUERY,
     {
-      variables: { baseId: baseIdInput && baseIdInput[1] || "0" },
+      variables: { baseId },
+    },
+  );
+
+  // fetch options for actions on boxes
+  const [runActionOptionsForBoxesView] = useLazyQuery<ActionOptionsForBoxesViewQuery>(
```

Comment:
remove these queries

Reply:
I think you meant the ones in the routes themselves, we want these to load first in the component tree.